src/translations/fr.json

```diff
@@ -326,5 +326,8 @@
     "SeriesResistanceText": "Résistance série",
     "SeriesReactanceText": "Réactance série",
     "ShuntConductanceText": "Conductance parallèle",
-    "ShuntSusceptanceText": "Susceptance parallèle"
+    "ShuntSusceptanceText": "Susceptance parallèle",
+
+    "PermanentCurrentLimitText": "IMAP",
```

Comment:
IST ? to be confirmed ...

Reply:
It's IMAP in the US, will confirm this with POs...

Reply:
Ok @ChristineHesse seems to be OK with 'IST', edited!